### PR TITLE
Part 13. Config for OpenRTB2 Blocking module 

### PR DESCRIPTION
**Pull request by 4lexvav**
Module PRD: https://docs.google.com/document/d/1TpB_I-Oft0LpZuKvVsaLAozOe1u52tpPGkB_jZfnMXo/edit?usp=sharing

### CODE REVIEW

**Comment by 4lexvav** [comment]
[Create ortb2blocking module skeleton](https://trello.com/c/jc3dmrfV/24-create-ortb2blocking-module-skeleton)

File: modules/prebid/ortb2blocking/config.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+}
+
+type Override struct {
+	IsActive bool
+	Ids      []int
+	Names    []string
+}
+
+func (o *Override) UnmarshalJSON(bytes []byte) error {
+	var d interface{}
```
Comment (AlexBVolcy) [inline modules/prebid/ortb2blocking/config.go]: Nitpick: Could we re-name this variable to be more descriptive?

File: modules/prebid/ortb2blocking/config.go [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+	if err := json.Unmarshal(bytes, &d); err != nil {
+		return err
+	}
+
+	switch v := d.(type) {
```
Comment (AlexBVolcy) [inline modules/prebid/ortb2blocking/config.go]: Nitpick: Could we re-name this variable to be more descriptive?

File: modules/prebid/ortb2blocking/config.go [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+type Override struct {
+	IsActive bool
+	Ids      []int
+	Names    []string
+}
+
+func (o *Override) UnmarshalJSON(bytes []byte) error {
+	var d interface{}
+	if err := json.Unmarshal(bytes, &d); err != nil {
+		return err
```
Comment (AlexBVolcy) [inline modules/prebid/ortb2blocking/config.go:116]: Nitpick: Would it be possible to add a test where the json unmarshal fails and returns an error?

**Review by AlexBVolcy — COMMENTED** [review]
Looks good, some small comments

File: modules/prebid/ortb2blocking/config.go (hunk 3 above)
Comment (4lexvav) [reply modules/prebid/ortb2blocking/config.go:116]: Sure, I added a new test

**Review by AlexBVolcy — APPROVED** [review]
LGTM

**Review by hhhjort — APPROVED** [review]
LGTM

**Comment by 4lexvav** [comment]
PR created in upstream repo

**Pull request closed without merging**